### DISCUSSION TITLE
License not availavble in Kibana

### DISCUSSION THREAD

**Discussion by 3isenHeiM** [Security-Onion-Solutions/securityonion#9486] [Unsupported Versions] [answered]
Hi everyone,

Since a couple of weeks, the kibana of my distributed architecture shows an error message about the license not being available.

![2022-12-30 15_31_19-https___srv03x-somgr01 pierre be_kibana_app_dashboards#](https://user-images.githubusercontent.com/26417172/210081981-40aec476-6e07-4567-ae89-732b0722853e.png)

The manager runs fine on the latest version (2.3.190), and `so-status` returns everything clear.

What am I missing? Thanks!

**Comment by dougburks** [comment, marked as answer]
Have you tried restarting Kibana?
```
sudo so-kibana-restart
```

Have you checked Kibana logs for clues?
https://docs.securityonion.net/en/2.3/kibana.html#diagnostic-logging

**Reply by 3isenHeiM** [reply]
A restart of Kibana appeared to solve the issue since I haven't witness it since.
